### DISCUSSION TITLE
What skin do you like?

### DISCUSSION THREAD

**Discussion by fredvs** [mse-org/mseide-msegui#79] [Q&A]
Hello.

What template of widget-set would you like to mimic with MSEgui?
It would be great to have more demos in mseuniverse of different custom-skins.

There are already the custom skins of cefou and me.
https://github.com/mse-org/mseuniverse/tree/main/skins

If possible, add a screenshot of some template you like (form Windows, MacOS, Linux and other systems).

Thanks.

**Comment by rchastain** [comment]
Good idea Fred.

By the way, I never used skins. What is the difference with using **tfacecomp** and **tframecomp**?

**Comment by fredvs** [comment]
>By the way, I never used skins.
>What is the difference with using tfacecomp and tframecomp?

Skins was the last feature Martin worked on.
It is partially complete and constitutes a sort of “centralizer” for the use of tfacecomp and tframecomp.

I also only use tfacecomp and tframecomp and assign it to each component, individually, not yet using skins.
With the "skin" feature, you can assign for each component-type the tfacecomp and tframecomp to use + other customization.
(That's what I understood, maybe I'm missing things).

It is a nice feature for beginners.
